yarnpkg/yarn
# fix(cli): allow binLinks to be set in .yarnrc (#4925)

**Summary**

Fixes #4925. Allow setting `bin-links false` in `.yarnrc`. Currently setting this option doesn't have any effect.

**Test plan**

* Create a package with a `.yarnrc` file containing the line `bin-links false`
* Run `yarn add gulp-cli` (or any package with binaries)
* Verify that no bin links were created in `node_modules/.bin`
